src/core_plugins/kibana/public/dashboard/panel/_dashboard_panel.scss

```diff
@@ -12,6 +12,7 @@
     flex: 1 1 100%;
     height: auto;
     z-index: 1;
+    background-color: inherit;
```

Comment:
I would add a comment to the end of the line saying that this fixes print layout reports, so it doesn't get removed (again?) :)

Reply:
:/ I didn't notice this comment until after I squashed + merged!